src/visual.ts

```diff
@@ -464,7 +459,7 @@ export class DotPlot implements IVisual {
 
         this.clearCatcher = appendClearCatcher(this.svg);
 
-        const axisGraphicsContext: Selection<any> = this.svg
```

Comment:
why removing types?

Reply:
I refactored to use actual types instead of `Selection<any>`.